src/components/ui/progress-circle.jsx

```
@@ -0,0 +1,100 @@
+import React from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)